libs/stats/odc/stats/io.py

```diff
@@ -247,13 +246,14 @@ def _ds_to_thumbnail_cog(self, ds: xr.Dataset, task: Task) -> List[Delayed]:
                                 crs=CRS.from_epsg(task.geobox.crs.to_epsg()))
 
         if task.product.preview_image_ows_style:
-
-            
             try:
                 thumbnail_cog = self._get_thumbnail(ds, task, task.product.preview_image_ows_style, input_geobox, odc_file_path)
                 thumbnail_cogs.append(thumbnail_cog)
             except AttributeError:
                 _log.error(f"Cannot parse OWS styling: {task.product.preview_image_ows_style}.")
+            except ImportError:
+                _log.error('Please run python -m pip install "odc-stats[ows]" to setup environment.')
```

Comment:
I think that if someone has requested an OWS style preview image in their configuration, it should error out at this point instead of only printing a message. It's the type of thing that is easy to get caught out by when running things for real.

Reply:
Change the code: Still throw the ImportError, but append more information: https://github.com/opendatacube/odc-tools/pull/392/commits/0d4b05e5375069f0dee1cd4a4f8080b86a79f3d2.